### DISCUSSION TITLE
How to convert to AIFF

### DISCUSSION THREAD

**Discussion by jrefano** [enzo1982/freac#372] [Audio Format Discussion]
I can't figure out how to output AIFF with any of the encoders.  Is it possible?

**Comment by enzo1982** [comment]
To convert to AIFF, select _SndFile Output Component_ as the encoder and then open the encoder settings. There you can set the output format to AIFF.

**Comment by jrefano** [comment]
Amazing, I went through a bunch but didnt try that encoder.  Thank you!!
